ogc-rs/src/runtime.rs

```diff
@@ -1,10 +1,8 @@
-//! The ``runtime`` module of ``ogc-rs``.
-//!
-//! This module implements runtime functions and allocators
-//! required for ``no_std`` on the Wii.
-//!
-//! Most of the functions defined here are modified functions from
-//! the helpful project [water](https://github.com/lemarcuspoilus/water).
+// This module implements runtime functions and allocators
+// required for ``no_std`` on the Wii.
+//
+// Most of the functions defined here are modified functions from
+// the helpful project [water](https://github.com/lemarcuspoilus/water).
```

Comment:
I'm not fully sure, but these comments will not show up on generated docs, which can be annoying for the end user if they don't have a source copy open.

Reply:
I believe this was something you wrote. Added them to to docs.

Reply:
Actually, I think I moved it from somewhere earlier. Should I still add to docs?

Reply:
Sure, if possible reverting it to the current would work since it uses ``//!`` for the comments instead.